ush/forecast_postdet.sh

```diff
@@ -217,7 +217,7 @@ FV3_out() {
   echo "SUB ${FUNCNAME[0]}: copying output data for FV3"
 
   # Copy configuration files
-  if [[ "${RUN}" == "gfs" || "${RUN}" == "gefs" ]]; then
+  if [[ "${RUN}" == "gfs" || "${RUN}" == "gdas" || "${RUN}" == "gefs" ]]; then
```

Comment:
At this point, we're just leaving out the enkf RUNs. Maybe we should just always do this?

Reply:
Good point, I had a similar thought and then didn't change it.  Before we leave out enkf, @CatherineThomas-NOAA do we want this for enkf too or no?

Reply:
I think we should save the enkf fcst configs as well since their settings are slightly different than the deterministic.  The files are small so it shouldn't be a space issue.

Reply:
The ensemble members are going to overwrite each other if we remove the wrap of conditionals.
Would just one ensemble member be enough or do all `NENS` members need to copy their configuration files?  The seeds are different based on the member number.
Let us know what the preference is, and we can address it in an future PR if it will take a little time to think.

Reply:
@aerorahul One member should be enough.

Reply:
Right now I've added the ~=enkf logic.  @CatherineThomas-NOAA i'll work with you offline to save the enkf ones? If we can, we'll add it to this PR, if not, we'll add it to a future update.

Reply:
COMOUT_CONF is defined with member numbers in the directory structure, so we wil not be over-writing each one.  Since these are very small files, I've just removed the if-statement.  If we want to walk this back a bit, if there's a if control variable we could only do this for the one.